bluebrain/repo-bluebrain/packages/py-pip-tools/package.py

```diff
@@ -3,24 +3,20 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+from spack.package import *
 
 
 class PyPipTools(PythonPackage):
     """pip-tools keeps your pinned dependencies fresh."""
 
     homepage = "https://pip-tools.rtfd.io"
-    pypi = "pip-tools/pip-tools-6.10.0.tar.gz"
+    pypi = "pip-tools/pip-tools-6.12.3.tar.gz"
 
-    # FIXME: Add a list of GitHub accounts to
-    # notify when the package is updated.
-    # maintainers = ['github_user1', 'github_user2']
+    version("6.12.3", sha256="480d44fae6e09fad3f9bd3d0a7e8423088715d10477e8ef0663440db25e3114f")
 
-    version("6.3.0", sha256="1835a1848bdfb22b2b6e5d10d630844ff5ee15e24b6c3bf92319c76f205d347f")
-
-    depends_on("py-click@7:")
-    depends_on("py-setuptools@45:", type="build")
-    depends_on("py-setuptools-scm+toml@6.2:", type="build")
-    depends_on("py-pip@20.3:")
-    depends_on("py-build")
-    depends_on("py-wheel")
+    depends_on("py-click@8:", type=("build", "run"))
+    depends_on("py-setuptools@63:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@63:", type=("build", "run"))
```

Reply:
Why was this resolved?

This is in `pyproject.toml`:
```
dependencies = [
  # direct dependencies
  "build",
  "click >= 8",
  "pip >= 22.2",
  # indirect dependencies
  "setuptools", # typically needed when pip-tools invokes setup.py
  "wheel", # pip plugin needed by pip-tools

]
```
Seems a clear run dependency to me...

https://github.com/jazzband/pip-tools/blob/6.12.3/pyproject.toml#L37-L47

Reply:
I have an impression that both the changes I suggested were undone, possibly by the force push?